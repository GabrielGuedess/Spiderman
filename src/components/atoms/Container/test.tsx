import { renderWithTheme } from 'utils/tests/helpers';

import { Container } from '.';

describe('<Container />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(<Container />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
