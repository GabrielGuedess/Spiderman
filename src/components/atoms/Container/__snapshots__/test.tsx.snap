// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['<Container /> should render the heading 1'] = `
.c0 {
  max-width: 144rem;
  margin: 0 2rem;
}

@media (min-width:1170px) {
  .c0 {
    margin: 0 3.2rem;
  }
}

@media (min-width:1440px) {
  .c0 {
    margin-left: auto;
    margin-right: auto;
  }
}

<div
  class="c0"
/>
`;
